prime-router/src/main/kotlin/transport/GAENTransport.kt

```diff
@@ -41,7 +41,7 @@ import java.io.ByteArrayInputStream
  * See [Google API for Exposure Notification](https://https://developers.google.com/android/exposure-notifications/exposure-notifications-api) for details on GAEN.
  * See [Issue API](https://https://github.com/google/exposure-notifications-verification-server/blob/main/docs/api.md#apiissue) for the details on the issue API ReportStream calls.
  */
-class GAENTransport : ITransport, Logging {
+class GAENTransport(val httpClient: HttpClient? = null) : ITransport, Logging {
```

Comment:
I don't understand this change. httpClient is only used once below and it is ignore and overwritten?